tests/sympc/tensor/replicatedshare_tensor_test.py

```diff
@@ -306,7 +306,7 @@ def test_ops_public_mul_integer(get_clients, parties, security):
     assert (result.reconstruct() == (secret * value)).all()
 
 
-@pytest.mark.parametrize("parties", [3, 5, 7])
```

Comment:
Q: Those changes are for speed-up?

Reply:
Yeah for speed up. I have addressed this as a general comment. Let me know what you think.